econplayground/templates/main/featuredgraph_list.html

```diff
@@ -55,7 +55,11 @@
 
                             <a role="button"
                                class="btn btn-sm btn-danger mr-1"
+<<<<<<< HEAD
```

Comment:
Looks like there's some leftover stuff from a merge to clean up here

Reply:
That's strange, It said it was ready to merge. I'll recheck and push featuredgraph_list